src/components/VendorSalesPage/VendorSalesPage.js

```diff
@@ -162,6 +173,49 @@ class VendorSalesPage extends Component {
                 <Input addonBefore={prefixSelector} style={{ width: '100%' }} placeholder="Enter valid phone number" />
               )}
             </Form.Item>
+            <Form.Item 
+              label="Establishment Year" 
+            >
+              {getFieldDecorator('establishedYear', {
+                rules: [{ required: true, message: 'Please select an year!' }],
+              })(
+                <Select placeholder="Select an Year">
+                  {Years}
+                </Select>
+              )}
+            </Form.Item>
+            <Form.Item label="E-mail">
+          {getFieldDecorator('email', {
+            rules: [
+              {
+                type: 'email',
+                message: 'The input is not valid E-mail!',
+              },
+              {
+                required: false
+              },
+            ],
+          })(<Input />)}
+          </Form.Item>
+          <Form.Item
+              label="Register your website"
```

Comment:
Change this to `Link your website`.
Also change habitual residence to shop address.

Reply:
Done